portfolio/src/main/java/com/google/sps/data/CommentUtility.java

```diff
@@ -104,31 +122,29 @@ public static void postComment(String collectionID, HttpServletRequest request)
     commentEntity.setProperty("name", name);
     commentEntity.setProperty("comment", comment);
     commentEntity.setProperty("timestamp", timestamp);
+
+    // Save comment image if it exists
+    String imageUrl = getUploadedFileUrl(request, "image");
+
+    System.out.println(imageUrl);
```

Comment:
This line looks like test code. Please remove

Reply:
Fixed it.